src/main/java/securbank/models/Account.java

```diff
@@ -0,0 +1,205 @@
+package securbank.models;
+
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import javax.persistence.CascadeType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import org.joda.time.LocalDateTime;
+
+/**
+ * 
+ * 
+ * @author Madhu Illuri
+ *
+ */
+
+
+@Entity 
+@Table(name = "Account")
+public class Account {
+	
+	
+	/** map ManyToOne to many side of the relationship */
+	
+	@ManyToOne(cascade = CascadeType.ALL)
+	@JoinColumn(name = "userId", nullable = false)
+	private User user;
+	
+	/** Account number is unique. */
+	@Id
+	@NotNull
+	@GeneratedValue(generator = "uuid2")
+	@GenericGenerator(name = "uuid2", strategy = "uuid2")
+	@Size(min = 5, max = 15)
```

Comment:
@madhuilluri here length and size of UUID doesn't match.
I don't think you need to specify size here and length in the next line
also, our account won't be UUID, it would be auto increment number.